packages/app/src/components/Root/Root.tsx

```diff
@@ -71,23 +69,30 @@ const SidebarLogo: FC<{}> = () => {
   );
 };
 
-const Root: FC<{}> = ({ children }) => (
+type SidebarItem = {
+  icon: typeof SvgIcon;
+  route: string;
+  label: string;
+};
+type SidebarProps = {
+  sidebarItems: SidebarItem[];
+  children: ReactNode;
+};
+
+const Root: FC<SidebarProps> = ({ sidebarItems, children }: SidebarProps) => (
   <SidebarPage>
     <Sidebar>
       <SidebarLogo />
       <SidebarSpacer />
       <SidebarDivider />
-      <SidebarItem icon={HomeIcon} to="/" text="Home" />
-      <SidebarItem icon={AccountCircle} to="/login" text="Login" />
+      {sidebarItems.map(d => (
+        <SidebarItem icon={d.icon} to={d.route} text={d.label} />
```

Comment:
Need a `key` prop in iterations

Reply:
Thanks for that. If I had a penny for everytime I just glossed over that one. :man_facepalming: